printer-creality-cr10v3-bltouch.cfg

```diff
@@ -0,0 +1,229 @@
+#Stock Creality CR-10 V3 with BL-Touch and runout filament sensor
```

Comment:
Add information how to build the firmware. Also include the board type and revision

Reply:
Done.
The printer comes with a Creality 3D v2.5.2 8-bit mainboard with an ATMega2560. The information was appended to the comment.